### DISCUSSION TITLE
Amiga performance very poor

### DISCUSSION THREAD

**Discussion by ghost** [rg35xx-cfw/Koriki#43] [Q&A]
Anyone managed to at least match GarlicOS performance on Amiga emulation? I tried various options (threaded video etc) but I can't make basically anything run well (Lotus Turbo Challenge 2 is my favourite test bench)

**Comment by acmeplus** [comment]
Have you tried puae2021 as the core? In my experience it runs fairly well. Haven't tried Lotus Turbo Challenge 2 though (although I spent hours and hours playing that ages ago....).

**Comment by ghost** [comment]
Thanks, I'll take a deeper look. I had some trouble Koriki remembering my changes to the setup, but we'll see ;)
